.github/workflows/run-cctp-tests.yml

```diff
@@ -0,0 +1,82 @@
+name: PR approved workflow
+
+on:
+  issue_comment:
+    types:
+      - created
```

Comment:
We should remove this `issue_comment`

Reply:
We want to be able to run cctp tests by adding a comment `/run-cctp-tests` in PRs